signing/signer.go

```diff
@@ -18,9 +18,10 @@ type Domain byte
 const (
 	ATX Domain = 0
 
-	BALLOT = 2
-	HARE   = 3
-	POET   = 4
+	PROPOSAL = 1
```

Comment:
looks strange that we didn't use 1 before

Reply:
@fasmat do you know why? 

maybe unrelated: 
staticcheck complains
`SA9004: only the first constant in this group has an explicit type (staticcheck)` 
if i don't leave a empty line.

Reply:
i think we had beacon as 1, and then @fasmat implemented change to split it into different domains